apps/dev-recruiters/src/components/modules/NewJoinPageComponent/Drag and Drop/draganddrop.tsx

```diff
@@ -0,0 +1,225 @@
+
+  //uncommented this
+  const [showUploadModal, setShowUploadModal] = useState(false);
+  const [isUploading, setIsUploading] = useState(false); // Uploading state
+  const maxSizeInMB = 25;
+  const maxSizeInBytes = maxSizeInMB * 1024 * 1024;
+  const allowedExtensions = /(\.doc|\.pdf|\.jpg|\.jpeg|\.png)$/i;
+  const [uploadError, setUploadError] = useState('');
+  const [canButVis, setCanButVis] = useState(true);
+  const portfolioUploadformData = new FormData();
+
+  //uncommented above lines
+  const [dragActive, setDragActive] = useState<boolean>(false);
+  let uploadedids = '';
+  const fileUpload = async (inFiles) => {
+    let uploadStatus = true;
+    setIsUploading(true);
+    const response = async () => {
+      if (!allowedExtensions.exec(inFiles[0].name)) {
+        setUploadError(
+          'The file you have chosen is not a valid file type. Please upload only .doc, .pdf, .jpg, .jpeg, and .png (Max 25MB)'
+        );
+        setIsUploading(false);
+        uploadStatus = false;
+        return false;
+      }
+      // Check file size
+
+      if (inFiles[0].size > maxSizeInBytes) {
+        setUploadError(
+          'The file you have chosen is not a valid file type. Please upload only .doc, .pdf, .jpg, .jpeg, and .png (Max 25MB)'
+        );
+
+        setIsUploading(false);
+        uploadStatus = false;
+        return false;
+      }
+      return true;
+    };
+    const responseResult = await response();
+
+    if (responseResult === true) {
+      portfolioUploadformData.append('files', inFiles[0]);
+
+      const postResult = await axios
+        .post(
+          `${process.env.NEXT_PUBLIC_STRAPI_URL}/googledrive/`,
+          portfolioUploadformData
+        )
+        .then((responseBody) => {
+          onFilesUploaded(responseBody.data);
+          setUploadFiles(responseBody.data);
+          setCanButVis(true);
+          setShowUploadModal(false);
+          setIsUploading(false);
+          uploadStatus = false;
+        });
+      if (!uploadStatus) return postResult;
+    }
+  };
+
+  const handleFileSelectChange = async (
+    event: ChangeEvent<HTMLInputElement>
+  ) => {
+    let sampleOutput = {
+      id: '17LV9EHZPGehHMvL86RdV1gmuV8VXL9fa',
+      name: 'Energy.jpg',
+      mimeType: 'image/jpeg',
+      parents: ['1jN1_Crat6nkpakD0BZsE3xKAIkJ26NE2'],
+      webViewLink:
+        'https://drive.google.com/file/d/17LV9EHZPGehHMvL86RdV1gmuV8VXL9fa/view?usp=drivesdk',
+    };
+    const inputFiles = Array.from(event.target.files);
+    fileUpload(inputFiles);
+  };
+
+  function dropHandler(e: any): void {
+    e.preventDefault();
+    // e.stoppropagation();
+    setDragActive(false);
+    if (e.dataTransfer.files && e.dataTransfer.files[0]) {
+      setSelectFiles((prevState: any) => [
+        ...prevState,
+        e.dataTransfer.files[0],
+      ]);
+    }
+    fileUpload(e.dataTransfer.files);
+  }
+
+  function dragLeaver(e: any): void {
+    e.preventDefault();
+    //e.stoppropagation();
```

Comment:
is there a specific reason to leave this commented code in place? could it be deleted?